cryptolib/cryptolib.go

```diff
@@ -3,20 +3,28 @@ package cryptolib
 import (
 	"encoding/json"
 	"io/ioutil"
+	"log"
 	"net/http"
 	"strconv"
-	"strings"
+	"sync"
+	"time"
 
 	"github.com/JoeParrinello/brokerbot/messagelib"
 )
 
+var (
+	mu          sync.Mutex
+	priceFeeds  []*PriceFeed
+	lastUpdated time.Time
```

Comment:
Should this be renamed (or a comment added to it) saying this is the last time "priceFeeds" was updated (in case in the future we have things that have different update schedules?)

Reply:
I'd rename it at the point that we add more variables.